src/main/java/de/tum/in/www1/artemis/repository/ExamRepository.java

```diff
@@ -37,8 +37,8 @@
     @EntityGraph(type = LOAD, attributePaths = { "registeredUsers", "exerciseGroups", "exerciseGroups.exercises" })
     Optional<Exam> findWithRegisteredUsersAndExerciseGroupsAndExercisesById(Long examId);
 
-    @EntityGraph(type = LOAD, attributePaths = { "studentExams" })
-    Optional<Exam> findWithStudentExamsById(Long examId);
+    @EntityGraph(type = LOAD, attributePaths = { "studentExams", "exerciseGroups", "exerciseGroups.exercises" })
```

Comment:
Have you seen the comment below in line 50? :see_no_evil:

Please do not load as many objects at the same time, because this will most likely lead to crashes in exams with many participating students

Reply:
Sorry, I had overlooked the comment. I reversed this change.